src/plugins/template/tests/functional/op_reference/interpolate.cpp

```diff
@@ -640,6 +640,21 @@ std::vector<InterpolateV4TestParams> generateParamsForInterpolate_v4_linear_onnx
                 NearestMode::ROUND_PREFER_FLOOR},
             {1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f},
             {1.0f, 4.0f},
+        },
+        {   "linear_onnx.resize_downsample_scales_linear_align_corners_channel_last",
+            Shape{1, 2, 4, 1},
+            {1, 2},
+            Shape{1, 1, 2, 1},
+            {0.6f, 0.6f},
+            {1, 2},
```

Comment:
This test case is okay, but it would be good to add few more tests, 
especially with the last dimension not equal to `1`.

Reply:
Hi Mitruska, I found an issue when I tried to do this. I noticed that even though I deliberately mess the test cases, changing the output array to a wrong one, when re-make ov_template_func_tests and run ./ov_template_func_tests --gtest_filter="*Interpolate*", it always shows RUN, OK. I would expect to see some error messages since I manually changed the test case setting to be incorrect. Could you help me with this situation?